fmp_data/lc/__init__.py

```
@@ -82,80 +72,80 @@
-    vector_store = EndpointVectorStore(
-        client=client,
-        registry=registry,
-        embeddings=embeddings,
-        cache_dir=cache_dir,
-        store_name=store_name,
-    )
+) -> Any:  # EndpointVectorStore
+    """Create a new vector store."""
+    if not is_langchain_available():
+        raise ImportError(
+            "LangChain dependencies required. Install with: pip install 'fmp-data[langchain]'"
```

Ruff (0.11.9)

Line too long (94 > 88)

(E501)